src/development/tools/devtools/console.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Using the Debug console
+description: Learn how to use the DevTools console to xxx.
```

Comment:
The "to xxx" might not be understood by everyone. I think just dropping it is fine:

```suggestion
description: Learn how to use the DevTools console.
```